orangecontrib/text/tests/test_preprocess.py

```diff
@@ -1,20 +1,22 @@
 import os
 import unittest
-from orangecontrib.text.preprocess import Preprocessor, Stemmer, Lemmatizer
+from orangecontrib.text.corpus import Corpus, get_sample_corpora_dir
+from orangecontrib.text.preprocess import Preprocessor, PorterStemmer, SnowballStemmer, Lemmatizer
 
 
 class PreprocessTests(unittest.TestCase):
     sentence = "Human machine interface for lab abc computer applications"
-    corpus = ["Human machine interface for lab abc computer applications",
-              "A survey of user opinion of computer system response time",
-              "The EPS user interface management system",
-              "System and human system engineering testing of EPS",
-              "Relation of user perceived response time to error measurement",
-              "The generation of random binary unordered trees",
-              "The intersection graph of paths in trees",
-              "Graph minors IV Widths of trees and well quasi ordering",
-              "Graph minors A survey",
-              ]
+    corpus = [
+        "Human machine interface for lab abc computer applications",
+        "A survey of user opinion of computer system response time",
+        "The EPS user interface management system",
+        "System and human system engineering testing of EPS",
+        "Relation of user perceived response time to error measurement",
+        "The generation of random binary unordered trees",
+        "The intersection graph of paths in trees",
+        "Graph minors IV Widths of trees and well quasi ordering",
+        "Graph minors A survey",
+        ]
 
     def test_preprocess_one_sentence_lowercase_stopwords(self):
```

Comment:
While you are at it, can you remove the `sorted` from this test? Now the preprocessor returns the tokens in order of appearance, it is not required any more.